As of writing, extensions for bundled gems are not installed without this patch.
We have a guard for that in build.sh and you can remove this without extra care.

--- a/common.mk
+++ b/common.mk
@@ -1375,7 +1375,6 @@
 
 refresh-gems: update-bundled_gems prepare-gems
 prepare-gems: $(HAVE_BASERUBY:yes=update-gems) $(HAVE_BASERUBY:yes=extract-gems)
-prepare-gems: $(DOT_WAIT) $(HAVE_BASERUBY:yes=outdate-bundled-gems)
 extract-gems: $(HAVE_BASERUBY:yes=update-gems)
 
 update-gems$(gnumake:yes=-sequential): PHONY
